app/src/app/components/contents/packets/PacketDetails.tsx

```diff
@@ -14,8 +16,11 @@ export const PacketDetails = () => {
       <PacketHeader packetName={packetName ?? ""} packetId={packetId ?? ""} displayName={packet?.displayName} />
       {longDescription && <p>{longDescription}</p>}
       <Separator className="mx-1" />
```

Comment:
For consistency with the sibling pages, I think we should drop this Separator when there is no longDescription displayed. So, move it into the conditional `<p>` in the previous line.